### PR TITLE
[spoteus] Add set-current-spot method

### PR DESCRIPTION
**Pull request by tkmtnt7000**
This PR enables users to execute 
```bash
rosservice call /spot_behavior_manager_server/reset_current_node_id "current_node_id: '<node>'"
``` 
from roseus interface.
This interface is useful when Spot stops moving midway.

- Usage
```
(send *ri* :set-current-spot "eng2_7FElevator")
```

### CODE REVIEW

File: jsk_spot_robot/spoteus/spot-interface.l [hunk 1]
```diff
@@ -318,6 +331,10 @@
           nil
         (send msg :data))
       ))
+  (:set-current-spot (node_id)
+    "set current node id"
```
Comment (mqcmd196) [inline jsk_spot_robot/spoteus/spot-interface.l]: I don't  know why but sometimes eus returns the error when string type data was set after method name, so please use comment out or fix like https://github.com/sktometometo/jsk_robot/pull/19

**Review by mqcmd196 — CHANGES_REQUESTED** [review]

**Comment by mqcmd196** [comment]
I don't know whether it has problem with function name.

File: jsk_spot_robot/spoteus/spot-interface.l (hunk 1 above)
Comment (tkmtnt7000) [reply jsk_spot_robot/spoteus/spot-interface.l]: #19 is saying something like below...?
```
(:set-current-spot
  "documentation"
    (args) 
    ...)      ;; return error
```
```
(:set-current-spot
   (args)
   "documentation" ...) ;; ok
```
If you are talking about the same thing as in the example given above, then I think the current writing style should be fine...

**Comment by tkmtnt7000** [comment]
> I don't know whether it has problem with function name.

I am concerned about function name which resembles `:get-current-spot`.
And I am just wondering if that function name directly describes its function.

File: jsk_spot_robot/spoteus/spot-interface.l (hunk 1 above)
Comment (mqcmd196) [reply jsk_spot_robot/spoteus/spot-interface.l]: oh, I misunderstood, yeah, it's no problem. sorry...

**Review by mqcmd196 — APPROVED** [review]

**Pull request merged**